packages/js/components/src/product-image/index.tsx

```diff
@@ -54,36 +76,4 @@ const ProductImage = ( {
 	);
 };
 
-ProductImage.propTypes = {
```

Comment:
Since we've removed this, we can also remove the `import PropTypes from 'prop-types';`. 🙂 

https://github.com/woocommerce/woocommerce/blob/f70a5f9e2a4a80f25f4890fcf79ae5152bc08b2f/packages/js/components/src/product-image/index.tsx#L5

Reply:
Thank you for catching it 👍 

Updated in https://github.com/woocommerce/woocommerce/pull/36305/commits/521e50f0a9803bcb547506e14498a779400bec3f